VenafiTppPS/Code/Public/Get-TppCertificate.ps1

```diff
@@ -121,19 +151,44 @@ function Get-TppCertificate {
 
         $params.Body.CertificateDN = $Path
 
-        if ( $PSBoundParameters.ContainsKey('IncludePrivateKey') ) {
+        if ($IncludePrivateKey.IsPresent) {
 
             # validate format to be able to export the private key
             if ( $Format -in @("Base64", "DER", "PKCS #7") ) {
```

Comment:
this was my bad, but hoping you don't mind changing it.  can you please change base64 to Base64 (PKCS #8)?  I had the wrong format.  thanks :)

Reply:
Not a problem @gdbarron , I will change it to

```powershell
if ( $Format -in @("Base64 (PKCS #8)", "DER", "PKCS #7") ) {
```
and check them all in at the same time as the changes below.